java/ql/src/semmle/code/java/frameworks/SnakeYaml.qll

```diff
@@ -27,7 +28,10 @@ class SafeSnakeYamlConstruction extends ClassInstanceExpr {
  * The class `org.yaml.snakeyaml.Yaml`.
  */
 class Yaml extends RefType {
-  Yaml() { this.getASupertype*().hasQualifiedName("org.yaml.snakeyaml", "Yaml") }
+  Yaml() {
+    this.getASupertype*()
+        .hasQualifiedName(getAShadedPackage("org.yaml.snakeyaml.constructor"), "Yaml")
```

Comment:
Was this changed on purpose, shouldn't it be the following? (might require formatting)
```suggestion
        .hasQualifiedName(getAShadedPackage("org.yaml.snakeyaml"), "Yaml")
```